subprojects/docs/src/samples/user-input/publishing-credentials/common/buildSrc/src/main/java/com/example/MavenRepositoryStub.java

```diff
@@ -0,0 +1,50 @@
+package com.example;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.ServerSocket;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.anyUrl;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.notFound;
+import static com.github.tomakehurst.wiremock.client.WireMock.ok;
+import static com.github.tomakehurst.wiremock.client.WireMock.put;
+import static com.github.tomakehurst.wiremock.client.WireMock.unauthorized;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+
+public class MavenRepositoryStub {
```

Comment:
I put it in `buildSrc` if we want to discourage the buildSrc classes on buildscript classpath - perhaps I should bundle this differently?